app/src/main/java/com/keylesspalace/tusky/util/twittertext/Regex.java

```diff
@@ -0,0 +1,348 @@
+// Copyright 2018 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Tusky changed: slight adaptions for Mastodon compatibility
+
+package com.keylesspalace.tusky.util.twittertext;
```

Comment:
I refrained from coverting these to Kotlin because in Java it is easier to compare them with upstream or the Mastodon app.